library/src/main/java/com/novoda/downloadmanager/lib/DownloadThread.java

```diff
@@ -97,7 +100,7 @@ public DownloadThread(Context context,
                           DownloadsUriProvider downloadsUriProvider,
                           DownloadsRepository downloadsRepository,
                           NetworkChecker networkChecker,
-                          DownloadReadyChecker downloadReadyChecker) {
+                          DownloadReadyChecker downloadReadyChecker, TarFileTruncator tarFileTruncator) {
```

Comment:
Move this to next line to keep proj convention (or at least this class conv :tongue:)